src/assets/code-screen.svg

```diff
@@ -0,0 +1,9 @@
NgwwqGlQW9eL08E8m7elAWzZspnH38oj454TN756BMJYL06MjQYKxfMl7fe+Jv07z9A/vvPf5G6jZvccTRwVvZfnl01bTlEYOWCeTIBtq1cmTZlbQ5BK7Yb0+5ffe01dZ/ruHewCZZR7EW5gyvsq3v5in+d9lIQ1gVODXyf6b2Z8pPmSN71DDDg9voWLDkZMWKEcqzgGJzqT3AMC+bNky3I8rl69ZrL+ycE2SyPP/a49OzdW6XL8linwYetiRMnSFLSaQiPWpMmQdD+GDxokAyGjgYJFJ0xMEtm1apVkoHzWGWAOO2np+3XpXMXeWLUaFz7HNwHcyXxVBJ0VKyvT1VgQ8KxT+8+8uz3fwDi8JCMG/e5Kit6mK9xDvR4OEkxYRAwCHgeAu0hTr4GNuRlibYoFed3rQmDgBMEKoTY4IodiQ0TBoGKRIAuJ6nnzkr8ti2yAgJ9TE3/9KuJ0hxWj6rkBKubCcjOWDB7JmxbA+WdDz6SNu07GNvWirwo99l2Oq7ZgYQ9MnXSBKEGiomyIVAXqe0vvvSSUMvBaaSg3Gc3Mp0o1GUndEkCw8s3WLyDm0n1sI6wbdXIdiq8LYU3L0lB5n7JzTpu2y2qpjeEsG93uJz0Atmgo6LOlfmDcB5Zh1Kzixcv2Z6jqCSE6f8UBiXh4DRugkRNw3lWg3BI2LvXktDw9vaSunXqCh/mBg8ZoiVsSocZlpxsQLYM9RVMlI4AyaIQlA21bNlKunTtqvRjFi9aKHtxffLyrUubSm+1fPcgMUfbVpaC9UQ2IYnAd//5tiqRciVeW769qLrWTiMjpR30Q0wYBAwCnodAZGSk+q7kAoDO9y2RCg0NU/bWj2noUXkewmbERQg4f+otOsLJv+A0CozAhhOkzD5lRICaCwvmzJJETACaQGX5F7/5rbRFqls1TFAyMDleBEvXRNi20vqRtq3tsYLnixp9E+6JAAmN1SuWyaF9e+U4XABM3B8CLIdwuvp7EiU+B6ENQVLjEsq1LLNjMHH09qsl1Wu1vmPbWqOOVgcLcxIl/2qKFFw+aVtywjpcPvzQrYXlGmEa5SApcLc4dPCQ7MIY6B7hquSkcaNGqjynNQhQ2tDpxH7cn/thrbp//wHhCnTJ4ANbQ5SZ9OwFC91WrWHh6ty6NhNlV4d4HVByQpcOpt+aKB2BAOhmdMLne7eu3YDZTdmBsp3j+Oy3uj6lt1b+e5B0IdFYZOXLlcdlIOIPU0Mjzz1Il/If9b0tGmLjXkzMFoOApyDA73ZqZZC41y31bNasqRIP5TONbvapp+BrxvktAhVCbMAPRfJNxsa3KJvfyg0BOp0smjtbZk6dInH9B8orv/6NNGzSTHyxypuPlbkZE8bLUqzU9ejZU57/yYvStHmU+OPBF4tjJtwQgVxM3uZBE2XThvWSDiswy0m1G/bbnbrUMSZUiQweOvGtsOcN1OiXhuVVTMznzp0rR0AkXUZ2jN3+LDnxq9sZtq1wOvEJdGzbqjC6lSl5F7bK7etpUnjbfqLetEkTGYLMhtbR0RKsIdjJyf/yZcskISEBLhKXbAkN9iUUq+Usa2kDfR0SnjpxKvGUrFy5QhKh0UA3FSusaNVK21YKg9ZC+05LTriCxZKTrXCboR3eLTz8mSgdAWbykMwYBFtUioJOnzYVJSen3IbQ4Ahq1PCDbetQOOvEQVj2oMxEuSSzfZj142mRZLKPPO2Sm/EaBL5BgMQGv68zsXiiT2w0k2SQ/fzc5Ge9CYOAKwQqhNi4jYwNU4jiCnbzmi4ChcgAupydJdNBXBw8sF/+4z//JO26dFOTB67S7d66Wf70xz9AhK2D/OWNN2Hb2hpkhmEzdHGulP3x4XALX1D7du2Qt/73r5VyyoftJDVreElURKD816stJDDAW2YvTZHixAYn3lwdKRmcRNMudNnSJbJp8xaloVFyH/U33jvVvf3Fr1aUSL2+lrvYbmTJCUiMwpRVcgsuJ3ZBS1WuZMdBUDOuf3+73e7ZzlUbEjdbt2xRNbt2Ti08kBkUJEpY1jJ6zJh72nK1gavpPM+cWbNk736Uz1jgyfZJaNCpZcyTT7pq7juv8frwIe0QymaWr1guKSmp33nd/GGPAO8bZt2MgoYGCW1en0PIfnCX4IoitSQ6ocxp9JgnlXbGJ9B+OgMxXitCzF36XdH9IClowiBgEPBMBEhEk9jgAoRuNMHCBzM9+H1siA1d9Dxv/4ohNkwZiufdSRU44nxMMPbA4vDLTz+Gw0m0/N///bvUuesqcByp2wtRksIPyz/9v9elS4+e4uvvX4G9MU3fLwJH4HCyculi2Lduu9+mPO744EC4nHQOkR6dQqVHx1rFxn8vibcLLh0jRo78Zh86QhyFdetmZAfwIcEy7pacVAuKEO9arZTLieV+NhsLrybL7atnJD/rKMgN6zR7PuBwYkpL1aLsBpvm7tlMHRDqTzC7ISU11bbchtkSLAmhhgZdYZit4TRI/pzCyn/Cnj0SDwytXE5IaLCMhZPXriBN6kPk1Gmw5OQE3FN24DON/xrbVmfI0Ra1KcoOe0OfollEBAjuA7J23Vq5hrIrdwgSGrTv7YiSyGg4ceXie2vc55+pjCinZWHuMI6K6gPLrEjsOM1kqqh+mHYNAgaBqkGA38NlITb42cpMNxIbJgwCpSFQIcSGKUMpDXbzug4C21CmsHThAnn+pZelO8QEfTAxEpBnm9aulnkQBu3Ws7c8/9OfSV0NC0Wd85t9yw+BxSChVixZLBegg2DCOQK+PtVlaN9w6dM1VDq2Cb7nQL7u5VUN4lzf5spRSJNBMoBkBkU1+VBht2rs5Rso3qGwbQ1sghx6PQ0NuXZW8rKOScHVs9DQsH744MNJJCakffv1U7ZtOumol6BLsB1EAMU609PToE2Qfw8GRRvqQJuDgqBMfW0EAkUnToD42QHB4SNHjn5T01v8+CK9hP7IMKEOSCMNDY2rV68KbWFJlpCUMQ9pxZEt/XcKTz7y6Ahl2frlF
```

Comment:
These seem to be SVG containers with embedded base64-encoded PNG bitmap files, rather than the source vector graphics?

Reply:
Unfortunately, Pri doesn't have the original image files. It would be better to revert back to PNG as they are smaller and can probably get the size down further. Maybe best for the flatter images like `sign-send`

Reply:
@pridesai7 can we ask the designer to send us the source vector files exported as SVG?